init.groovy.d/bitbucket-config.groovy

```diff
@@ -58,39 +64,55 @@ import java.util.Base64;
 	else{
 	
 		def home_dir = System.getenv("JENKINS_CONF")
-		def properties = new ConfigSlurper().parse(new File("$home_dir/simpleci.conf").toURI().toURL())
 		
-		properties.bitbucketConfig.each() { configName, serverConfig ->
-						
-			//Jenkins Credentials
-	
-			String id = java.util.UUID.randomUUID().toString()
-			// Decode password using base64
-			byte[] valueDecoded = Base64.getDecoder().decode(serverConfig.password);
-			Credentials c = (Credentials) new UsernamePasswordCredentialsImpl(CredentialsScope.GLOBAL,id, serverConfig.username, serverConfig.username, new String(valueDecoded))
-			SystemCredentialsProvider.getInstance().getStore().addCredentials(Domain.global(), c)
+		def file 					= new File("$home_dir/simpleci.conf")
+		int i=1;
+		// check if the file exists at the location. if not wait for it for 2 minutes. Repeat this process for 5 times.	
+		while(!file.exists() && i<6){
+			out.println ("I am a test info log")
+			i++;
+			Thread.sleep(120000)
+		}
+		//if the file not read even after 10 minutes, print the error message else read the file and continue.
+		if(i==6){
+			out.println ("Config file not present at the location. Please generate the file and try again later")
```

Comment:
out.println ("Config file not present at the location $home_dir/simpleci.conf. Please generate the file and try again later")